config/presets/ldp/response-writer.json

```diff
@@ -20,10 +20,10 @@
         },
         {
           "@type": "LinkRelMetadataWriter",
-          "LinkRelMetadataWriter:_headerMap": [
+          "LinkRelMetadataWriter:_linkRelMap": [
             {
-              "LinkRelMetadataWriter:_headerMap_key": "http://www.w3.org/1999/02/22-rdf-syntax-ns#type",
-              "LinkRelMetadataWriter:_headerMap_value": "type"
+              "LinkRelMetadataWriter:_linkRelMap_key": "http://www.w3.org/1999/02/22-rdf-syntax-ns#type",
+              "LinkRelMetadataWriter:_linkRelMap_value": "type"
```

Comment:
@joachimvh It was about time that we were switching to Components.js 🙂
This feature worked in the integration tests, but not in the actual server.